Addition of note (#9545)
MicrosoftDocs/windows-itpro-docs: windows/security/information-protection/bitlocker/ts-bitlocker-decode-measured-boot-logs.md

## Use PCPTool to decode Measured Boot logs

> [!NOTE]

> > PCPTool is a visual studio solution and need to build the executeable before using this tool.

Reviewer: A few details: Visual Studio is the name of a Microsoft software package, usually capitalized.
```suggestion
> PCPTool is a Visual Studio solution, but you need to build the executable before you can start using this tool.
```
There are several ways this sentence can be phrased. I am not entirely sure which phrasing is better, especially in terms of the Microsoft Writing Style Guide.

Feel free to suggest if this sentence should be phrased with a requirement in focus, or if it should be less "pointing at" the end user.

Reviewer 2: @illfated - I see we did the review at the same time, but I am happy to go with your suggestion and have deleted mine.

Reviewer: Thank you for confirming. Feels like I would have promoted your suggestion if I had seen it, but never mind.

> > PCPTool is a visual studio solution and need to build the executeable before using this tool.

Reviewer 2: ```suggestion
> PCPTool is a visual studio solution and needs to build the executable before it can be used.
```
